BoxContentSDK/BoxContentSDK/OAuth2/BOXAuthorizationViewController.m

```diff
@@ -16,6 +16,7 @@
 #import "BOXContentClient+Authentication.h"
 #import "BOXOAuth2Session.h"
 #import "BOXAppUserSession.h"
+#import "BOXDispatchHelper.h"
```

Comment:
In fact, we shouldn't have a dependency on dispatch helper in here... Is there a reason to do it this way?